docs/onboarding.md

```diff
@@ -26,6 +26,24 @@ Install dependencies
 npm install
 ```
 
+## 0. Link wallet to organization
+
+Attaches wallet to an organization. This is to later view and manage your projects from the developer dashboard.
+
+Run the following script
+
+_Requires environment variables to be set._
+
+- _`AUTH0_TENANT_URL`_
+- _`AUTH0_CLIENT_ID`_
+- _`AUTH0_AUDIENCE`_
+
+```sh
+npm run onboarding:link-wallet-to-organisation
+```
+
+This will open up a browser window where you can authorize your device and gain an access token. This will be used along side an IMX signature to authorize against the IMX Developer Dashboard.
```

Comment:
```suggestion
This will open up a browser window where you can authorize your device and gain an access token. This will be used alongside an IMX signature to authorize against the IMX Developer Dashboard.
```